Sources/DocCDocumentation/DocCDocumentation.docc/V5_API_Migration_guide.md

```diff
@@ -39,16 +40,15 @@ If you're using the Firebase SDK, you'll need to follow [these instructions](htt
 Version 5.0 of the SDK  deprecates the term "Observer Mode" (and the APIs where this term was used), and replaces it
 with `PurchasesAreCompletedBy` (either RevenueCat or your app).
 
-Version 5.0 of the SDK also introduces support for recording purchases made directly by your app calling StoreKit 2. 
+Version 5.0 of the SDK also introduces support for tracking purchases made directly by your app calling StoreKit 2.
 
-You can enable it when configuring the SDK:
+If you're using RevenueCat only to track purchases, and you have your own implementation of StoreKit to make purchases, you will need to explicitly configure the SDK.
+
+Add this configuration only if you previously had `observerMode: true` in your SDK initialization or your app has its own implementation of StoreKit to make purchases.
 
 | Version 4 | Version 5 |
 |------------|------------|
-| <pre lang="swift"><code>Purchases.configure(with: .builder(withAPIKey: apiKey)<br>  .with(observerMode: true)<br>  .build()</code></pre> | <pre lang="swift"><code>Purchases.configure(with: .builder(withAPIKey: apiKey)<br>  .with(purchasesAreCompletedBy: .myApp, storeKitVersion: .storeKit2)<br>  .build()</code></pre> |
-
-
-
+| <pre lang="swift"><code>Purchases.configure(with: .builder(withAPIKey: apiKey)<br>  .with(observerMode: true)<br>  .build()</code></pre> | <pre lang="swift"><code>Purchases.configure(with: .builder(withAPIKey: apiKey)<br>  // Set only if your app has its own implementation of StoreKit to make purchases.<br>   Select the version of StoreKit you're using.<br>  .with(purchasesAreCompletedBy: .myApp, storeKitVersion: /* Select .storeKit1 or .storeKit2 */)<br>  .build()</code></pre> |
```

Comment:
I've merged the SK1 and SK2 observer mode docs into one so it's less confusing, and also made the sample break compilation to make the developer have to explicitly select the version of storekit they are using.